v2g-liberty-cards/src/ping-card.ts

```diff
@@ -33,7 +39,11 @@ export class PingCard extends LitElement {
   public connectedCallback() {
     super.connectedCallback();
     this._connected = true;
-    this._startPinging();
+
+    // Add a delay to avoid snackbar flickering
```

Comment:
This means an extra 15 seconds is added before the system detects that the addon is not responding. I do not think that is desirable.
I propose to fix the flickering in some other way.

Reply:
True. In production this seems no problem to me but i can image this is too long for dev.
Maybe start pinging and and suppress the error until the first ping is successful or if it takes longer than this timeout?

Reply:
I now made the delay 1 second (instead of 15) and the unjustly showing of the error now is absent. A on second delay seems ok for dev as well.